Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift

```diff
@@ -65,15 +66,20 @@ class ConfirmYourBidPINViewController: UIViewController {
                         .updateUserCredentials(loggedInProvider)
                         .mapReplace(provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { provider -> Observable<BypassResults> in
                     return me
-                        .fulfillmentNav()
-                        .updateUserCredentials(loggedInProvider)
-                        .mapReplace(provider)
+                        .viewModel
+                        .checkForAdminCCBypass(bidDetails.auctionID, authorizedNetworking: provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { results -> Observable<Void> in
+                    // Check if we should bypass the CC requirement and move directly onto the next step.
+                    guard results.bypassCCRequirement == false else {
```

Comment:
I feel like the guard here kinda makes the logic a bit harder to follow, might be easier to just do an if